src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,26 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register(EMAIL, PASSWORD);
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Failed to register with such email and password", e);
+        }
```

Comment:
The exception handling for registration should provide meaningful feedback and handle exceptions appropriately. Consider logging the exception or providing more context in the error message.